src/mmds/src/lib.rs

```diff
@@ -65,6 +65,20 @@ fn build_response(http_version: Version, status_code: StatusCode, body: Body) ->
     response
 }
 
+// Make the URI a correct JSON pointer value.
+fn sanitize_uri(uri: &str) -> String {
```

Comment:
Would it make sense to make `uri` a String here and avoid the following line?

Reply:
The `uri` variable is a &str as we retrieve it here. I would still have to call to_string() somewhere else.
```
let uri = request.uri().get_abs_path();
```

Reply:
I know, the idea here was to call to_string() when calling `sanitize_uri()`, not inside of it; it was a matter of having one less code line.

Reply:
Done.